databases/week1/homework.sql

```diff
@@ -0,0 +1,87 @@
+-- Find out how many tasks are in the task table
+select
+    COUNT(id)
```

Comment:
You should create a new line per statement it's easier to read, e.g.
`SELECT COUNT(*) as NumberOfTasks`
`FROM task;`